index.js

```diff
@@ -1,22 +1,27 @@
 import express from 'express';
 import bodyParser from 'body-parser';
 import dotenv from 'dotenv';
-import routes from './routes/index';
+import swaggerUi from 'swagger-ui-express';
+import routes from './server/routes';
+import swaggerDoc from './hello-books-swagger.json';
 
 dotenv.config();
 const port = process.env.PORT || 4000;
 const app = express();
 
 const { log } = console;
 
+log(process.env.NODE_ENV);
```

Comment:
Great stuff @LukasChiama very clean code as usual, however, is there any particular reason why you're logging the environment to the console or is this just as a heads up to be aware of the environment we're working in?

Reply:
O boy! I totally missed that. I used it to confirm the setup was working properly and then forgot to delete before pushing. Thanks for catching this. I'll take it away immediately.